app/services/shop_service/update.rb

```diff
@@ -0,0 +1,37 @@
+# PATH: /app/services/shop_service/update.rb
+# rubocop:disable Style/ClassAndModuleChildren
+class ShopService::Update
+  attr_accessor :id, :name, :address, :current_user
+  def initialize(id, name, address, current_user)
+    @id = id
+    @name = name
+    @address = address
```

Comment:
The `ShopService::Update` service class is using manual validation which can be error-prone and hard to maintain. Consider using ActiveModel validations for better maintainability.